src/pybind/mgr/dashboard/controllers/nfsganesha.py

```diff
@@ -129,30 +122,21 @@ class NFSGaneshaExports(RESTController):
     @EndpointDoc("List all NFS-Ganesha exports",
                  responses={200: [EXPORT_SCHEMA]})
     def list(self):
-        '''
-        list exports based on cluster_id ?
```

Comment:
Like the CLI already does, I think we need to be able to list exports of a particular NFS cluster. It's weird to see the feature request mentioned as a doc string. So I agree we need to remove it. Is there a tracker ticket for this request?